ios/Podfile.lock

```diff
@@ -53,4 +53,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 70d9d25280d0dd177a5f637cdb0f0b0b12c6a189
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.14.3
```

Comment:
On real project we usually do not touch config files, just keep in mind that

Reply:
Ok